values-it/strings.xml

```diff
@@ -28,47 +28,47 @@
         <!--SUMMARIES-->
             <string name="explanation_dark_mode">Il tema scuro migliora la leggibilità in condizioni di luce scarsa</string>
-            <string name="summary_wifi_only">Scarica solo tramite Wi-Fi</string>
+            <string name="summary_wifi_only">Download solo tramite Wi-Fi</string>
             <string name="explanation_newest_version">Elimina vecchie versioni se una nuova variante simile è stata scaricata</string>
-            <string name="explanation_keep_packages">%1$d pacchetti simili più vecchi saranno conservati</string>
+            <string name="explanation_keep_packages">Saranno conservati %1$d pacchetti simili più vecchi</string>
             <string name="explanation_download_md5">Scarica il checksum MD5 per verificare l’integrità del pacchetto</string>
             <string name="explanation_download_versionlog">Scarica il registro versione per visualizzare il contenuto del pacchetto</string>
             <string name="summary_root_mode">Concedi l’accesso root per abilitare la modalità di installazione e l’azzeramento cache/dalvik</string>
             <string name="explanation_wipe_dalvik">Forza ri-ottimizzazione della cache al prossimo avvio. Impiega più tempo, ma aiuta a prevenire eventuali problemi.</string>
-            <string name="explanation_pref_install_warning">Mostra un avviso prima dell’installazione per evitare installazioni accidentali</string>
-            <string name="explanation_pref_delete_warning">Mostra un avviso prima di eliminare un pacchetto per evitare cancellazioni accidentali</string>
+            <string name="explanation_pref_install_warning">Visualizza un avviso prima dell’installazione per evitare installazioni accidentali</string>
+            <string name="explanation_pref_delete_warning">Visualizza un avviso prima di eliminare un pacchetto per evitare cancellazioni accidentali</string>
         <!--DIALOG-->
-            <string name="explanation_cleanup_now">Pulire ora i vecchi pacchetti??</string>
+            <string name="explanation_cleanup_now">Pulizia pacchetti obsoleti?</string>
         <!--FILECHOOSER (EXTERNAL LIBRARY)-->
             <string name="label_confirm">Conferma</string>
             <string name="label_select">Seleziona</string>
             <string name="label_create">Crea</string>
             <string name="text_freespace">%1$s liberi</string>
-            <string name="title_storage_chooser">Seleziona Memoria</string>
-            <string name="title_storage_chooser_internal">Memoria Interna</string>
-            <string name="label_select_external_storage">Seleziona Memoria Esterna</string>
+            <string name="title_storage_chooser">Seleziona memoria</string>
+            <string name="title_storage_chooser_internal">Memoria interna</string>
+            <string name="label_select_external_storage">Seleziona memoria esterna</string>
```

Comment:
In italy the grammar rules explain thet only first lettter of a phrase is in uppercase except for own name.